src/index.ts

```diff
@@ -239,6 +240,25 @@ task("starknet-compile", "Compiles Starknet contracts")
     .addFlag("disableHintValidation", "Allows compiling a contract with any python code in hints.")
     .setAction(starknetCompileAction);
 
+task("starknet-compile-cairo1", "Compiles Starknet cairo1 contracts")
+    .addOptionalVariadicPositionalParam(
+        "paths",
+        "The paths to be used for deployment.\n" +
```

Comment:
deployment?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/commit/e5836b358f557103a39b64cc449cdd59404125b9

Reply:
`The paths to be used for sources` -> An odd sentence; the paths are source files of contracts to be compiled.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/328/commits/97ff6b34b516e99cf938c3a254356402e446d088